src/main/kotlin/notary/endpoint/eth/EthRefundStrategyImpl.kt

```diff
@@ -133,9 +125,30 @@ class EthRefundStrategyImpl(
         }
     }
 
+    /**
+     * Check if [srcAccountId] has Ethereum withdrawal [address] in whitelist
+     */
+    private fun checkWithdrawalAddress(srcAccountId: String, address: String): Result<Boolean, Exception> {
+        val whitelistSetter = "whitelist_setter@notary"
+
+        return getAccountDetails(
+            irohaConfig,
+            keypair,
+            irohaNetwork,
+            srcAccountId,
+            whitelistSetter
+        ).map { details ->
+            val whitelist = details["eth_whitelist"]
+
+            if (whitelist != null)
```

Comment:
Koltin should have a better way to do that

Reply:
Well, explicit null check removes nullable reference and is not so bad in Kotlin. .?let{} operator doesn't allows else branch. If you have better approach, please share it.

Reply:
```
whitelist?.let{
    whitelist.split(",").contains(address)
} ?: run {
    false
}
```

Reply:
imho != null is more meaningful